packages/symfony-toggle/test/DependencyInjection/TestContainerFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pheature\Test\Community\Symfony\DependencyInjection;
+
+use Pheature\Community\Symfony\DependencyInjection\PheatureFlagsExtension;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+
+final class TestContainerFactory
+{
+    public static function create(CompilerPassInterface $compilerPass): ContainerBuilder
+    {
+        $containerBuilder = new ContainerBuilder();
+        $extension = new PheatureFlagsExtension();
+        $containerBuilder->registerExtension($extension);
+
+        $phpFileLoader = new YamlFileLoader($containerBuilder, new FileLocator());
+        $phpFileLoader->load(realpath(__DIR__ . '/../../config/config.yaml'));
+
+        $compilerPass->process($containerBuilder);
```

Comment:
Looks really good the use of this `TestContainerFactory`, the only drawback is that we're instantiating the real class, so that the unit test will access the filesystem to retrieve the YAML file, parsing it and loading it to the container  :/. But I think that we have more benefits in terms of test simplicity, so 👏🏽